lumibot/backtesting/polygon_backtesting.py

```diff
@@ -249,15 +249,23 @@ def get_chains(self, asset):
         option_contracts = {"SMART": {"TradingClass": None, "Multiplier": None, "Expirations": [], "Strikes": []}}
         contracts = option_contracts['SMART']  # initialize contracts
         today = self.get_datetime().date()
+        real_today = date.today()
 
         # All Contracts | to match lumitbot, more inputs required from get_chains()
-        polgon_contracts = self.polygon_client.list_options_contracts(
-            underlying_ticker=asset.symbol,
-            expiration_date_gte=today,
-            expired=True,  # Needed so BackTest can look at old contracts to find the expirations/strikes
-            limit=1000
-        )
-        for polygon_contract in polgon_contracts:
+        # If the strategy is using a recent backtest date, some contracts might not be expired yet, query those too
+        expired_list = [True, False] if real_today - today <= timedelta(days=31) else [True]
```

Comment:
Copied from chat:
Polygon has a 50k limit of the number of candles it will return for one query.  For a crypto asset on a 24/7 market that is ~34days of data.  So I chopped it down to "1 month" and used that as the chunk size